appcheck/firebase-appcheck/ktx/ktx.gradle

```diff
@@ -18,7 +18,7 @@ plugins {
 }
 
 firebaseLibrary {
-    releaseWith project(':appcheck:firebase-appcheck')
+    libraryGroup "appcheck"
```

Comment:
I don't see `appcheck/firebase-appcheck` in this PR. Is it missing from this library group?